app/scripts/map/cartodb-vis-directive.js

```diff
@@ -6,8 +6,31 @@
 (function () {
     'use strict';
 
+    var orgPopupTemplate = [
+        '<div class="popup">',
+        '<div ng-if="loading" class="spinner">Loading...</div>',
+        '<div ng-if="!loading">',
+        '<div ng-if="rows && rows.length && rows.length > 1" class="popup-header">',
+        '<p>Found {{ rows.length }} results:</p>',
+        '</div>',
+        '<div class="popup-content">',
+        '<div class="popup-content-row" ng-repeat="row in rows">',
+        '<a href="/#/museum/{{row.ein_new}}/">{{ row.organization_new }}</a>',
+        '</div>',
+        '</div>',
+        '</div>',
+        '</div>'
+    ].join('');
+
+    var orgPopoverTemplate = [
+        '<div>',
+        '<p>{{ orgName }}</p>',
+        '<p ng-if="resultCount > 1">Found {{ resultCount - 1 }} additional locations. Click for details.</p>',
```

Comment:
"locations" isn't really what we mean here.  "organizations" or "results" would be better.  Also, I wonder about "Found", since it implies a subject (i.e. "[We] Found").  Could also go with something less active that doesn't raise implicit questions about whose voice the site is speaking in.  e.g. "[There are] 6 additional results at this location:"

Reply:
👍  will change. Way better.

Reply:
Could it state "## additional organizations at this location." ?